profanity_word_phrase_filter/README.md

```diff
@@ -1,19 +1,19 @@
 ## Custom profanity filter
 
-Most online platforms contain inappropriate language. Parent's want to be certain
+Most online platforms contain inappropriate language. Parents want to be certain
 that Story squad (aka scribble stadium) is offering a safe environment for their
-kiddos. In compliance to the children's Online Protection rule (COPPA)
+kiddos in compliance with the Children's Online Privacy Protection Rule (COPPA).
 https://www.ftc.gov/enforcement/rules/rulemaking-regulatory-reform-proceedings/childrens-online-privacy-protection-rule
 
-It's important for the submitted stories to be flagged as inappropriate if they 
-contain any profanity. The stories will still be viewed by a human eye to make
-sure that the flagging is warranted. 
-The previous cohort worked on using the available libraries like profanity-filter
-and profanity_Check but they were not sufficient in flagging all words and newer
+It is important for submitted stories to be flagged as inappropriate if they 
+contain any profanity. The stories will still be reviewed by a live person to make
+sure the flagging is warranted. 
+The previous cohort worked on using available libraries like profanity-filter
+and profanity_Check but these could not sufficiently flag all profane words, newer
 phrases and Scuntorpe like words.
 
-Using a list of profanity phrases in (bad_phrases.csv) and profanity words in (bad_single.csv).
+Using a list of profanity phrases in (bad_phrases.csv) and profanity words in (bad_single.csv);
```

Comment:
This whole paragraph seemed a bit unruly. I therefore provide the following sentence to simplify and clarify its meaning. 
"Using a list of profanity phrases in (bad_phrases.csv) and profanity words in (bad_single.csv), profanity_word_phrase_filters.py can be used on a string of text to identify and flag words and phrases as inappropriate."

Reply:
Your sentence definitely sounds better.  Checkout the changes.

Reply:
The changes you made were warranted.  I also think the following sentence is a bit clunky, 
"The stories will still be reviewed by a live person to make
sure the flagging is warranted." 
Maybe make the sentence say, 
"The stories will still be reviewed manually by a person to make
sure flagging, by the system, is warranted."

Reply:
Definitely agree with you, Jay. Checkout the changes made at you request.